src/components/SearchBox/__tests__/SearchBox-test.js

```diff
@@ -0,0 +1,347 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import SearchBox from '../SearchBox';
+
+const defaultProps = {
+  placeholder: '',
+  cssClasses: {
+    root: 'root',
+    form: 'form',
+    input: 'input',
+    submit: 'submit',
+    submitIcon: 'submitIcon',
+    reset: 'reset',
+    resetIcon: 'resetIcon',
+    loadingIndicator: 'loadingIndicator',
+    loadingIcon: 'loadingIcon',
+  },
+  templates: {
+    reset: x => x,
+    submit: x => x,
+    loadingIndicator: x => x,
+  },
+};
+
+describe('SearchBox', () => {
+  describe('Props', () => {
+    describe('cssClasses', () => {
+      test('sets all CSS classes', () => {
+        const wrapper = mount(<SearchBox {...defaultProps} />);
+
+        expect(wrapper.find('div').hasClass('root')).toEqual(true);
+        expect(wrapper.find('form').hasClass('form')).toEqual(true);
+        expect(wrapper.find('input').hasClass('input')).toEqual(true);
+        expect(
+          wrapper.find('button[type="submit"]').hasClass('submit')
+        ).toEqual(true);
+        // expect(
```

Comment:
We don't have access to this element since it's in the template.

Reply:
makes sense not to test it